packages/block-library/src/preformatted/styles.native.scss

```diff
@@ -0,0 +1,27 @@
+.wpBlockPreformatted {
+	padding-left: 16;
+	padding-right: 16;
+	padding-top: 12;
+	padding-bottom: 12;
+	background-color: $gray-light;
+	border-top-left-radius: 4;
+	border-top-right-radius: 4;
+	border-bottom-left-radius: 4;
+	border-bottom-right-radius: 4;
```

Comment:
These are fine, but you can also use CSS shorthand properties to write it more concisely:

```css
padding: 12px 16px;
border-radius: 4px;
```

Reply:
I try to use them but I don't think they work with our scss to style transformers.

Reply:
Ok, after some more testing I managed to make it work, it looks you need to use px if you try this:
`12 16` or `12pt 16pt` it doesn't work.

Will pixels be alright taking in account the multiple screen densities?

Reply:
I don't think the CSS transform is that smart about pixel densities, and it just seems to remove the units when using `px`:

```
> const transform = require('css-to-react-native-transform').default
undefined
> transform('.test { padding: 12px; margin-left: 12; margin-right: 12pt; margin-top: 12% }')
{ test:
   { paddingTop: 12,
     paddingRight: 12,
     paddingBottom: 12,
     paddingLeft: 12,
     marginLeft: 12,
     marginRight: '12pt',
     marginTop: '12%' } }
```

After that step, [the docs say](https://facebook.github.io/react-native/docs/height-and-width):

> All dimensions in React Native are unitless, and represent density-independent pixels